integration/worker/dns_test.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/concourse/concourse/integration/internal/dctest"
+	"github.com/concourse/concourse/integration/internal/flytest"
+)
+
+func TestDNSProxyyEnabled(t *testing.T) {
```

Comment:
Nit: 
```suggestion
func TestDNSProxyEnabled(t *testing.T) {
```

sim for Disabled